package.json

```diff
@@ -2,34 +2,36 @@
   "name": "@lifeomic/prim",
   "version": "1.0.0",
   "description": "Prim is a customizable TypeScript atomic styles framework for React Native.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
```

Comment:
Am I correct that these can be removed because of the updated new `postbuild` script?

Reply:
I _think_ so. Since I'm `cp`-ing the `package.json` into `dist/`. `index.js` & `index.d.ts` are right there at the root. Just trying to be like `chroma-react`.

Reply:
Got it, so then we'd publish from `dist/`. Sounds good!